hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/util/Shell.java

```diff
@@ -86,7 +86,15 @@
    */
   @Deprecated
   public static boolean isJava7OrAbove() {
-    return true;
+    return isJavaSpecAtLeast(7);
```

Comment:
no, this can stay true. We don't support old stuff any more

What would be good is to have a test in {{TestShell}} which calls the new API operation & sees that it is consistent with its expectations (i.e on trunk it must be 8+)

Reply:
Ok, reverted.